### PR TITLE
Remove unused parameter

### PR DESCRIPTION
**Pull request by nohwnd**

### CODE REVIEW

File: src/Analyzers/MSTest.Analyzers.CodeFixes/TestClassShouldBeValidFixer.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
             diagnostic);
     }
 
-    public static async Task<Document> FixClassDeclarationAsync(Document document, SyntaxNode root, ClassDeclarationSyntax classDeclaration, CancellationToken cancellationToken)
+    public static async Task<Document> FixClassDeclarationAsync(Document document, SyntaxNode _, ClassDeclarationSyntax classDeclaration, CancellationToken cancellationToken)
```
Comment (Evangelink) [inline src/Analyzers/MSTest.Analyzers.CodeFixes/TestClassShouldBeValidFixer.cs]: Let's remove the parameter instead please

Comment (nohwnd) [reply src/Analyzers/MSTest.Analyzers.CodeFixes/TestClassShouldBeValidFixer.cs]: I tried that, but the testing fixture that consumes this method requires this method signature.

Comment (Evangelink) [reply src/Analyzers/MSTest.Analyzers.CodeFixes/TestClassShouldBeValidFixer.cs]: Huh? We should not have any test fixture using that (only line 50).

Comment (nohwnd) [reply src/Analyzers/MSTest.Analyzers.CodeFixes/TestClassShouldBeValidFixer.cs]: yeah you are right, I've misread what is happening.

**Review by Evangelink — APPROVED** [review]

**Pull request merged**